src/pytorch_metric_learning/utils/logging_presets.py

```diff
@@ -365,7 +365,12 @@ def record_group_name(self, tester, split_name):
         )
         if reference_sets_label == query_set_label:
             reference_sets_label = "self"
-        return f"{base_record_group_name}_{query_set_label}_vs_{reference_sets_label}"
+        record_group_name = f"{base_record_group_name}_{query_set_label}_vs_{reference_sets_label}"
+        if record_group_name in self.record_keeper.hash_map.values():
```

Comment:
I think we can get rid of this if statement

Reply:
Yes, true